docs/source/quickstart/99_FAQ.md

```diff
@@ -17,3 +20,45 @@ make local_env_create LOCAL_ENV_OPTS="-K -e cifmw_local_env_vm_graphics=none"
 
 The important part here is the `-e cifmw_local_env_vm_graphics=none` - it will instruct virsh to start the VM
 without any graphics. This will NOT prevent `virsh -c qemu:///system console` accesses.
+
+### Debug control plane deployment failures
+If the control plane deployment fails or timeout, you can try debug the failure with the command
+outputs below:
+
+To check the virtual machines that are running/active:
+
+```Bash
+$ sudo virsh list
```

Comment:
`sudo` isn't useful here, it's better to pass the `-c qemu:///system` option

Reply:
updating